bin/proving-service/README.md

```diff
@@ -105,16 +105,22 @@ miden-proving-service start-worker
 To start the proxy service, you will need to run:
 
 ```bash
-miden-proving-service start-proxy --prover-type transaction [worker1] [worker2] ... [workerN]
+miden-proving-service start-proxy --prover-type transaction [worker1],[worker2],...,[workerN]
```

Comment:
Given that the list of workers is optional, it feels like an option-based approach (i.e., with `--workers` flag) could be more appropriate here - e.g.,:
```
miden-proving-service start-proxy --prover-type transaction --workers [worker1],[worker2],...,[workerN]
```